packages/react-tei/src/tags/TableNotes.tsx

```diff
@@ -0,0 +1,27 @@
+import Stack from "@mui/material/Stack";
+
+import type { DocumentJson } from "../parser/document.js";
+import { Value } from "./Value.js";
+
+export function TableNotes({ notes }: TableNotesProps) {
+	return (
+		<Stack
+			gap={1}
+			sx={{
+				"& p": {
+					padding: 0,
+					margin: 0,
+					color: "text.secondary",
+				},
+			}}
```

Comment:
I don't think the note should  style its children,
the note tag can contain a lot of possible things, including table.

By having it style the decendant p we could endup with unintended consequences IMO